torchtitan/components/metrics.py

```diff
@@ -163,6 +163,25 @@ def close(self) -> None:
         if self.wandb.run is not None:
             self.wandb.finish()
 
+class LoggerHandler(BaseLogger):
```

Comment:
according to torchtitan convention, this sounds should be Container than Handler, see e.g. https://github.com/pytorch/torchtitan/blob/main/torchtitan/components/optimizer.py#L39
```suggestion
class LoggerContainer(BaseLogger):
```

Reply:
TB and W&B usually have different purposes. TB is usually the choice for live monitoring of jobs while running. W&B for grouped analysis after results have finished. TB is also the usual workflow for pulling data and plotting data offline. But mainly, enabling W&B would silently override the setting for TB and turn it off seemed like the main error.